### PR TITLE
[+] Additions to React Roadmap

### PR DESCRIPTION
**Pull request by nirvikagarwal**
<hr>

## Description 📜

Made some additions to the React Roadmap.

- ES6 features
- Anatomy of a react application
- Use of React Developer Tools

<hr>

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev (React, Angular, Vue)
<hr>

### CODE REVIEW

**Review by mukeshmjn — APPROVED** [review]
Looks fine to me @anandhu720 u can check it once and finally merge!

**Comment by Jaya3112** [comment]
please assign me this issue

File: Frontend_Web_Development_React_Angular_Vue/React/Readme.md [hunk 1]
```diff
@@ -8,17 +8,20 @@
 - [ ] JSX 
 - [x] Babel 
 - [ ] ES6 template literals 
+- [ ] ES6 features (destructuring, higher order array methods, arrow functions, spread operator etc.)
```
Comment (the-pro) [inline Frontend_Web_Development_React_Angular_Vue/React/Readme.md:11]: It is a part of JavaScript domain and is not needed here

File: Frontend_Web_Development_React_Angular_Vue/React/Readme.md [hunk 2]
```diff
@@ -8,17 +8,20 @@
...
 ### Week 2 
 - [ ] Components 
 - [ ] Class components 
-- [ ] Function components 
-- [ ] import,export,modules 
+- [ ] Functional components 
+- [ ] import, export, modules 
 - [ ] React Props vs State 
 - [ ] Mapping data to components 
+- [ ] Using the React Developer Tools
```
Comment (anandhu720) [inline Frontend_Web_Development_React_Angular_Vue/React/Readme.md:24]: Add it in week 1,like installing React developer tols

**Review by anandhu720 — CHANGES_REQUESTED** [review]
Some minor changes are required

**Comment by mukeshmjn** [comment]
@anandhu720 check it if it looks fine this time?

**Comment by nirvikagarwal** [comment]
Made the suggested changes, kindly review

**Review by anandhu720 — APPROVED** [review]

**Pull request merged**